pkg/entrypoint/server_test.go

```diff
@@ -129,34 +129,41 @@ var _ = Describe("entrypoint status API test", func() {
 		for _, tt := range testCases {
 			By(tt.name)
 			func() {
-				cancel := startRunner(apiAddr, tt.input)
+				cancel := startRunner(tt.input)
 				defer cancel()
 				for _, expected := range tt.expected {
 					Eventually(getStatus, 10, 0.5).Should(Equal(&expected))
 				}
 			}()
+			Eventually(func() error { _, err := getStatus(); return err }, 10, 0.5).Should(HaveOccurred())
```

Comment:
Strictly speaking, It does not wait for the termination of the runner. For example, if the runner returns a malformed result, `getStatus` will return error due to `json.Unmarshal` error.

We should write a dedicated function that checks the termination.